package.json

```diff
@@ -100,7 +100,8 @@
     "selenium-standalone": "6.12.0",
     "style-loader": "0.19.1",
     "webpack": "3.10.0",
-    "webpack-node-externals": "1.6.0"
+    "webpack-node-externals": "1.6.0",
+    "yup": "^0.24.1"
```

Comment:
I think fix version is recommended like: `"yup": "0.24.1"`
Also, we might need to add its license into our `NOTICE.txt` but maybe @jasonblais can confirm the requirement for it.

Reply:
You are right again :) I have changed the yup package to be fixed version. About NOTICE.txt, we don't bundle yup in the builds, is only for testing purpose, like enzyme (and enzyme license is not in the NOTICE.txt). And enzyme is a MIT license too.